src/Microsoft.DotNet.Cli.Utils/CommandResolution/ProjectToolsCommandResolver.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.DotNet.Cli.Utils
 {
     public class ProjectToolsCommandResolver : ICommandResolver
     {
-        private static readonly NuGetFramework s_toolPackageFramework = FrameworkConstants.CommonFrameworks.NetStandardApp15;
+        private static readonly NuGetFramework s_toolPackageFramework = FrameworkConstants.CommonFrameworks.NetCoreApp10;
```

Comment:
Nuget restores tools only for `netcoreapp` starting with https://github.com/NuGet/NuGet.Client/commit/9573f19ce94c84078e0d0e4f128bb24de7bfe140 so having two frameworks in CLI won't help anything. To make `netstandardadpp` tool work you just need to add `"import":"netstandardadpp"` to tool section.